nmosnode/aggregator.py

```diff
@@ -73,19 +86,25 @@ def __init__(self, logger=None, mdns_updater=None):
         self._node_data = {
             'node': None,
             'registered': False,
+            'auth_client_registered': False,
             'entities': {
                 'resource': {
                 }
             }
         }
+        self._reg_queue = gevent.queue.Queue()
```

Comment:
Does moving these three lines not cause a problem? I'd have thought they'd depend upon the variables initialised below.